src/codegen/codegen_c_visitor.cpp

```diff
@@ -1028,16 +1028,6 @@ std::string CodegenCVisitor::get_parameter_str(const ParamVector& params) {
 }
 
 
-void CodegenCVisitor::print_channel_iteration_task_begin(BlockType type) {
-    // backend specific, do nothing
-}
-
-
-void CodegenCVisitor::print_channel_iteration_task_end() {
-    // backend specific, do nothing
-}
-
-
 void CodegenCVisitor::print_channel_iteration_tiling_block_begin(BlockType type) {
```

Comment:
@alkino : I believe we should get rid of `*channel_iteration_tiling_block*` stuff as well. I don't see any relevance/usefulnes of this.